onnx/defs/schema.h

```diff
@@ -889,6 +907,10 @@ class OpSchemaRegistry final : public ISchemaRegistry {
     // Key: domain. Value: <lowest version, highest version> pair.
     std::unordered_map<std::string, std::pair<int, int>> map_;
 
+    // Key: domain. Value: most recent release opset version. Note that
+    // the highest opset version may be ahead of the most recent release's opset version.
+    std::unordered_map<std::string, int> last_release_version_map_;
+
```

Comment:
can you also add an api to get the last release version number

Reply:
Interesting that this does not already exist. It seems more broadly usable, so let me create a separate "version.h" in onnx/onnx/common.